crates/bevy_render/src/camera/projection.rs

```diff
@@ -100,12 +100,31 @@ impl Default for Projection {
     }
 }
 
+/// A 3D camera projection in which distant objects appear smaller than close objects.
 #[derive(Component, Debug, Clone, Reflect, FromReflect)]
 #[reflect(Component, Default)]
 pub struct PerspectiveProjection {
+    /// The vertical field of view (FOV) in radians.
+    ///
+    /// Defaults to a value of π/4 radians.
     pub fov: f32,
+
+    /// The aspect ratio (width divided by height) of the viewing frustum.
+    ///
+    /// Bevy's [`camera_system`](crate::camera::camera_system) automatically updates this
+    /// value when the aspect ratio of the associated window changes.
     pub aspect_ratio: f32,
+
+    /// The distance from the camera in world units of the viewing frustum's near plane.
+    ///
+    /// Objects closer to the camera than this value will not be visible. Defaults to a value of
+    /// `1000.0`.
     pub near: f32,
+
+    /// The distance from the camera in world units of the viewing frustum's far plane.
+    ///
+    /// Objects farther from the camera than this value will not be visible. Defaults to a value of
+    /// `1.0`.
```

Comment:
Default value is actually 1000.0.

Reply:
Whoops, got tripped up by the different field order in the struct declaration and the `Default` impl :sweat_smile: